content/how-to-persist-app-data-across-os-updates.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How to Persist Application Data across Torizon OS Updates
+date: 2024-12-02
+author: Leonardo Graboski Veiga, Product Owner Torizon OS
+draft: false
+tags:
+    - Container
+    - Embedded
+    - Linux
+abstract: How to make sure your application data is not overwritten or erased by OS Updates
+image: ./types-of-mounts.webp
+---
+
+The Torizon OS is designed around the concept that different parties handle _applications_ and _OS_ development:
+
+- Applications: you, as a Torizon customer, develop and update your application on your development cycle
+- OS: Toradex, more specifically the Torizon OS Team, develops and updates the base OS, releasing it every quarter
+
+A question I sometimes get from customers is: how can I keep my application data when updating the base OS?
+
+## A Practical Answer
+
+**You must store the application data somewhere under the `/var` directory**. OSTree, the file system management tool used for OS Updates, keeps the `/var` directory intact during the updates.
+
+Therefore, the two standard Docker [storage methods](https://docs.docker.com/engine/storage/) are ok to use.
```

Comment:
Mention both by name so it's easier to the reader to understand the next paragraphs?